parameter/ConfigurableDomain.cpp

```diff
@@ -301,7 +301,7 @@ bool CConfigurableDomain::parseConfigurableElements(const CXmlElement& xmlElemen
         core::Results infos;
         if (!addConfigurableElement(pConfigurableElement, NULL, infos)) {
 
-            CUtility::asString(infos, strError);
+            utility::asString(infos, strError);
```

Comment:
This is has a completely different effect than it previously had. Same comment in all places below.